internal-packages/simple-cached-fetch/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "$schema": "https://json.schemastore.org/package.json",
+  "private": true,
+  "name": "@data-eden/simple-cached-fetch",
+  "version": "0.2.2-beta.1",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/data-eden/data-eden.git",
+    "directory": "internal-packages/simple-cached-fetch"
+  },
+  "license": "MIT",
+  "type": "module",
+  "exports": {
+    ".": {
+      "import": {
+        "types": "./src/index.ts",
+        "default": "./src/index.ts"
+      }
+    }
+  },
+  "main": "./dist/index.js",
+  "typings": "./dist/index.d.ts",
+  "scripts": {
+    "test": "vitest run"
+  },
+  "dependencies": {
+    "@data-eden/network": "0.2.2-beta.1",
```

Comment:
Should these be `*` to ensure they always pull the latest and thus if we release changes that break this package, we'll catch it when it happens?

Reply:
For example, this PR is already behind since `0.3.1` was released today.